### ISSUE TITLE
Pressing a Link from react-aria-component fails in mobile

### ISSUE THREAD

**Issue by andreslemusm** [adobe/react-spectrum#7171]
### Provide a general summary of the issue here

On mobile devices, the `Link` component does not respond to click events when the user taps on a child SVG element inside the `Link`.

### 🤔 Expected Behavior?

A click event should be triggered

### 😯 Current Behavior

Click events are ignored

### 🖥️ Steps to Reproduce

https://codesandbox.io/p/sandbox/react-aria-touch-6xj9lw

### Version

1.4.0

### What browsers are you seeing the problem on?

Chrome, Safari

### What operating system are you using?

macOS 15.0.1

**Comment by yihuiliao** [comment]
Hm..it's a little finicky sometimes but I am able to press on the link with the svg on my iPhone 13 iOS 17.6.1. Was there a specific device that you were seeing this on?

**Comment by andreslemusm** [comment]
> Hm..it's a little finicky sometimes but I am able to press on the link with the svg on my iPhone 13 iOS 17.6.1. Was there a specific device that you were seeing this on?
> 
> Edit: That said, my success rate when pressing a normal `<a>` tag around the svg was much better than when using Link so seems like a bug but not sure what's causing it

I updated the sandbox to include a third anchor using an <a> tag.

This video is on an iPhone 13 Pro iOS 18.0.1

https://github.com/user-attachments/assets/9e043710-42da-493d-97fc-53faa2012afa

Looks like the press works if you do it outside the SVG path, in the anchor's paddings

**Comment by yuxxeun** [comment]
+1
I'm facing the same problem on the ui library that uses `RAC` based.

**Comment by snowystinger** [comment]
I believe the easiest way to work around it is to put `pointerEvents: "none"` on the SVG. Could you give that a try?

If this just started happening, it may be related to [Mobile Item Selection Fails Starting from Version 3.22.3 of @react-aria/interactions](https://github.com/adobe/react-spectrum/issues/7157#top)

Otherwise, this issue may be resolved when we do https://github.com/adobe/react-spectrum/issues/1720

**Comment by andreslemusm** [comment]
> I believe the easiest way to work around it is to put `pointerEvents: "none"` on the SVG. Could you give that a try?
> 
> If this just started happening, it may be related to [Mobile Item Selection Fails Starting from Version 3.22.3 of @react-aria/interactions](https://github.com/adobe/react-spectrum/issues/7157#top)
> 
> Otherwise, this issue may be resolved when we do #1720

@snowystinger thanks for the workaround, it works! (updated the example to include it).

It just started happening, so it's probably related to #7157

**Comment by LFDanLu** [comment]
@andreslemusm We've just released a patch that contains https://github.com/adobe/react-spectrum/pull/7190 so closing this out for now, but definitely let us know if you are still running into issues!

**Comment by andreslemusm** [comment]
> @andreslemusm We've just released a patch that contains #7190 so closing this out for now, but definitely let us know if you are still running into issues!

Thanks for the fix! it is now working as expected without any workarounds
